src/evaluation_system/tests/plugin_manager_test.py

```diff
@@ -318,6 +318,43 @@ def test_get_history(dummy_settings, temp_user):
     assert all([g is not None for g in g2])
     assert g1[0] == g2[0]
 
+    import random
```

Comment:
I think we have a full coverage over [`get_history`](https://app.codecov.io/gh/FREVA-CLINT/freva/blob/main/src%2Fevaluation_system%2Fapi%2Fplugin_manager.py#L1010) and no new line has not been added in `plugin_manager.py`, so i think it makes more sense for more readability and simplicity of the code, the part of test which is related to `all-users` would be better to be removed

Reply:
I have a doubt there .

On the one hand, (as you pointed out) the core functionality has not changed, but on the other, taking a look at the [older version of the method](https://github.com/FREVA-CLINT/freva/blob/main/src/evaluation_system/api/plugin_manager.py#L1010) we never tested the case where `uid=None`, which is the whole point of the change. 
```python
    user = user or User()

    if plugin_name is not None:
        plugin_name = plugin_name.lower()

    return user.getUserDB().getHistory(
        plugin_name,
        limit,
        since=since,
        until=until,
        entry_ids=entry_ids,
        uid=user.getName(), # <----
    )
```

Reply:
I just commented out the `test_get_history_all` part and got a test coverage, and there was no change in `get_history` coverage without `get_history_all` part. the coverage tool evaluates the logic on a line-by-line basis. If part of a line is executed, it doesn't account for whether the rest of the line is used or not.